src/pages/Home.js

```diff
@@ -76,8 +76,27 @@ export default function Home(props) {
             value={inputValue}
             onChange={handleInputValue}
           />
+          <IconButton
+            type="submit"
+            icon={
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                class="h-6 w-6"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke="currentColor"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  stroke-width="2"
```

Comment:
Looks great! Now THAT's a chevron icon!

It looks like you copied the SVG version from [Heroicons](https://heroicons.com/). Which is fine, because all you have to do is make a couple changes to get rid of the console errors:

`class` -> `className`
`stroke-linecap` -> `strokeLinecap`
`stroke-linejoin` -> `strokeLinejoin`
`stroke-width` -> `strokeWidth`

OR you can just copy in the JSX version which does all that for you (which is the difference between the two).

Reply:
Ah! I had it in my mind that SVG files were the ones we were using. Easy fix 👍 

And yes, this is an actual chevron 😄